client/src/main.ts

```diff
@@ -68,27 +70,47 @@ async function startVue () {
   Vue.prototype.$USAStateCodes = USAStateCodes
   Vue.prototype.$xproMapping = mapping.xproMapping
 
+  // Initialize Sentry
+  if (window['sentryDsn']) {
+    console.info('Initializing Sentry...') // eslint-disable-line no-console
+    Sentry.init({
+      dsn: window['sentryDsn'],
+      integrations: [
+        // new Integrations.Vue({ Vue, attachProps: true }), // FUTURE maybe
+        new Integrations.CaptureConsole({ levels: ['error'] })
```

Comment:
Note 1: I'm afraid to enable the Vue integration as I think it will be too "noisy".

Note 2: The second integration captures all `console.error()` calls. As you write error handling, judge for yourself if the message should be captured by Sentry or simply logged to the console.